extensions/amp-brightcove/0.1/validator-amp-brightcove.protoascii

```diff
@@ -36,6 +36,13 @@ tags: {  # <amp-brightcove>
     name: "data-account"
     mandatory: true
   }
+  # <amp-bind>
+  attrs: { "[data-account]" }
```

Comment:
Does that mean that you expect folks to do something like:

`<amp-brightcove data-account="foo" [data-account]="...">`

Where the attribute is effectively specified twice?